neqo-transport/src/connection/mod.rs

```diff
@@ -3064,31 +3054,38 @@ impl Connection {
         }
     }
 
-    fn confirm_version(&mut self, v: Version) -> Res<()> {
-        if self.version != v {
-            qdebug!("[{self}] Compatible upgrade {:?} ==> {v:?}", self.version);
-        }
-        self.crypto.confirm_version(v)?;
-        self.version = v;
-        Ok(())
+    const fn has_version(&self) -> bool {
+        self.crypto.has_handshake_keys()
     }
 
+    /// Commit to a particular version.
+    ///
+    /// `packet_version` is set to `Some` when the client is calling,
```

Comment:
`packet_version` is neither an `Option`, nor is it set in the function below. Is this comment outdated?